Cargo.toml

```diff
@@ -167,15 +167,15 @@ warp = { version = "0.3", default-features = false }
 warp-embed = { version = "0.4", default-features = false }
 wascap = { version = "0.12", path = "./crates/wascap", default-features = false }
 wash-cli = { version = "0", path = "./crates/wash-cli", default-features = false }
-wash-lib = { version = "0.15", path = "./crates/wash-lib", default-features = false }
+wash-lib = { version = "0.16", path = "./crates/wash-lib", default-features = false }
```

Comment:
Are we ready to release these dependent crates? I ask because e.g. if we needed to do a patch bump of wash-cli, and we've changed the versions here but haven't published them, then publishing wash-cli would fail, right?

Reply:
Very true. I don't think we need to release them yet, but I do think that we need to update the versions in these crates because there's new functionality. Put another way, if the newly updated code for `wash-cli` tries to compile against `wash-lib 0.15`, it will fail.

With this method of development in the monorepo, I think we have to bump crates when making breaking changes or pay attention to the conventional commits leading up to a release. This change going in means wash-cli has to bump a minor version, so we can't publish a patch anyways unless we cherry pick this commit out. Does that make sense? I want to make sure I'm thinking about this the right way since we should all take the same actions when working on changes that affect multiple libraries in this repo.